### ISSUE TITLE
Unable to render visualisations in Jupyter Notebook

### ISSUE THREAD

**Issue by mshearer0** [moj-analytical-services/splink_demos#77]
Unable to render visualisations in Jupyter Notebook:

```
<VegaLite 4 object>

If you see this message, it means the renderer has not been properly enabled
for the frontend that you are using. For more information, see
https://altair-viz.github.io/user_guide/troubleshooting.html
```

Suggest:
```alt.renderers.enable('default')```

**Comment by henryf3** [comment]
Facing the same issue when try to use splink with jupyter notebook (using aws interactive sessions). It just shows the message:
`<splink.charts.VegaliteNoValidate object at 0x7f639faf6410>`

**Comment by RobinL** [comment]
Have you tried `alt.renderers.enable('default')`?

**Comment by henryf3** [comment]
@RobinL yes, i tried every mode available. I think it is related to the kernel Glue Pyspark, since in the regular kernel Python it works fine. Any suggestion would be welcome!
